clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerTestBuilder.java

```diff
@@ -61,7 +61,7 @@ public class ConsumerTestBuilder implements Closeable {
     static final int REQUEST_TIMEOUT_MS = 500;
 
     final LogContext logContext = new LogContext();
-    final Time time = new MockTime(1, 0, 0);
+    final Time time = new MockTime(0);
```

Comment:
I'm doing this on purpose - I don't think we want auto-tick for most of the time.  If we do I can revert this